message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -72,7 +72,7 @@
       </entry>
       <entry value="42002" name="MAV_CMD_SOLO_BTN_FLY_HOLD">
         <description>FLY button has been held for 1.5 seconds.</description>
-        <param index="1">Takeoff altitude.</param>
+        <param index="1" label="takeoff altitude" units="m">Takeoff altitude.</param>
```

Comment:
Cool. Description of the parent message could be improved, though I guess a re-implementation could change the behaviour of the button being held.